src/configModels.ts

```diff
@@ -44,6 +44,7 @@ export interface MqttConfiguration extends Record<string, unknown> {
    reject_unauthorized?: boolean;
    keepalive?: number;
    version?: number;
+   force_disable_retain?: boolean;
```

Comment:
I don't think this name is correct. In Zigbee2MQTT the `retain` option is actually set based on this setting, it has nothing to do with the QoS option.

Using the same name for a different setting does not sound okay to me. I think it's better to call it something like `disable_qos`.